docs/releases/2.0.txt

```diff
@@ -210,6 +212,36 @@ Database backend API
 
 * ...
 
+:attr:`AbstractUser.last_name <django.contrib.auth.models.User.last_name>` ``max_length`` increased to 100
+----------------------------------------------------------------------------------------------------------
+
+A migration for :attr:`django.contrib.auth.models.User.last_name` is included.
+If you have a custom user model inheriting from ``AbstractUser``, you'll need
+to generate and apply a database migration for your user model.
+
+If you want to preserve the 30 character limit for last names, use a custom
+form when creating a user or changing last names::
+
+    from django.contrib.auth.forms import UserCreationForm
+
+    class MyUserCreationForm(UserCreationForm):
```

Comment:
Some thinking is needed here. Unlike the `username` field, the `UserCreationForm` doesn't have `last_name` field. This needs to be reflected on the user change form instead.